mobify/platform-scaffold
# Check if push messaging is enabled before trying to click the dismiss…

… button

## Changes
- Wrap the click in if/else for cases when push messaging is not enabled/available, as is the case currently for SFCC

## How to test-drive this PR
- Verify there are no e2e test failures on https://circleci.com/gh/mobify/platform-scaffold/tree/check-if-push-is-present
